components/input.html

```diff
@@ -0,0 +1,139 @@
+                        <input type="text" placeholder="Enter name" aria-label="input-field-name">
+                    </div>
+
+                </div>
+                <div class="iframe-container">
+                    <iframe class="iframe input-field-frame" title="myFrame"
+                        src="https://carbon.now.sh/embed?bg=rgba%2874%2C144%2C226%2C1%29&t=material&wt=none&l=auto&width=680&ds=false&dsyoff=20px&dsblur=68px&wc=true&wa=true&pv=56px&ph=56px&ln=false&fl=1&fm=Fira+Code&fs=14px&lh=152%25&si=false&es=2x&wm=false&code=%253Cdiv%2520class%253D%2522input-group%2522%253E%250A%2520%2520%2520%2520%253Clabel%253EUsername%2520*%253C%252Flabel%253E%2509%2509%2509%2509%2509%2509%250A%2520%2520%2520%2520%253Cinput%2520type%253D%2522text%2522%2520placeholder%253D%2522Enter%2520name%2522%2520aria-label%253D%2522input-field-name%2522%253E%250A%253C%252Fdiv%253E"
+                        sandbox="allow-scripts allow-same-origin">
+                    </iframe>
+                </div>
+
+                <!-- Error/validation Input -->
+                <h2>Error/validation Input</h2>
+                <div class="avatar-container">
+                    <div class="input-group err">
+                        <label>Password *</label>
+                        <input type="text" placeholder="Enter password" aria-label="input-field-password">
+                        <div class="error-message">
+                            Wrong Password. Try again.
+                        </div>
+                    </div>
+                </div>
+                <div class="iframe-container">
+                    <iframe class="iframe input-frame" title="myFrame"
+                        src="https://carbon.now.sh/embed?bg=rgba%2874%2C144%2C226%2C1%29&t=material&wt=none&l=auto&width=680&ds=false&dsyoff=20px&dsblur=68px&wc=true&wa=true&pv=56px&ph=56px&ln=false&fl=1&fm=Fira+Code&fs=14px&lh=152%25&si=false&es=2x&wm=false&code=%253Cdiv%2520class%253D%2522input-group%2520err%2522%253E%250A%2520%2520%2520%2520%2520%2520%253Clabel%253EPassword%2520*%253C%252Flabel%253E%250A%2520%2520%2520%2520%2520%2520%253Cinput%2520type%253D%2522text%2522%2520placeholder%253D%2522Enter%2520password%2522%2520aria-label%253D%2522input-field-password%2522%253E%250A%2520%2520%2520%2520%2520%2520%253Cdiv%2520class%253D%2522error-message%2522%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520Wrong%2520Password.%2520Try%2520again.%250A%2520%2520%2520%2520%2520%2520%2520%253C%252Fdiv%253E%250A%253C%252Fdiv%253E"
+                        sandbox="allow-scripts allow-same-origin">
+                    </iframe>
+                </div>
+                <!-- Footer -->
+                <footer>
+                    <div class="prev">
+                        <i class="fas fa-angle-left"></i>
+                        <a href="./image.html">Image</a></div>
+                    <!-- <div class="next"><a href="">Alert</a>
+                        <i class="fas fa-angle-right"></i>
```

Comment:
Remove the commented html

Reply:
This I put for the next page when I add.